pipeline/extract_to_s3.py

```diff
@@ -1,14 +1,13 @@
 import os
 import logging
 import timeit
-from extract_generation import main as extract_generation
+from pipeline.extract_generation import main as extract_generation
```

Comment:
This will break the code if pushed as an image to the Cloud, Does this need to be here?